apps/dashboard/config/initializers/upgrade_to_2.1.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-Rails.application.reloader.to_prepare do
+Rails.application.config.after_initialize do
```

Comment:
This isn't really a part of this PR, but I just figured this is a better stage to do this work.